h2o-algos/src/test/java/hex/glm/GLMBasicTestBinomial.java

```diff
@@ -1589,6 +1589,46 @@ public void testBinomialIRLSMStandardize() {
       Scope.exit();
```

Comment:
@syzonyuliia Please have a brand new test for your work and not added to the current test.  You can call it GLMComputePValuesTest or something like that.  Use  your creativity.